### ISSUE TITLE
Borders around tiles at specific zoom levels

### ISSUE THREAD

**Issue by Nekzuris** [openstreetmap/iD#10589]
### URL

https://www.openstreetmap.org/edit#map=15/48.24613/7.00700

### How to reproduce the issue?

Use this custom layer over France and change the zoom level in small steps using the mouse wheel.
`https://data.geopf.fr/wmts?service=WMTS&format=image/png&version=1.0.0&request=GetTile&style=normal&tilematrixset=PM&tilematrix={z}&tilerow={y}&tilecol={x}&layer=IGNF_ELEVATION.ELEVATIONGRIDCOVERAGE.SHADOW`

Some zoom levels are fine, some show only horizontal or vertical lines, and some show a full grid.

I’ve experienced this issue for a long time on my phone with Chrome when I enabled "Show desktop version", Firefox is fine.
But now I also experience it on my computer (on Chrome and Firefox), but only with this custom layer. Note that the layer has transparency.

### Screenshot(s) or anything else?

![image](https://github.com/user-attachments/assets/fe41bf37-b6d4-447d-8498-227430af4ee0)
![Screenshot_20241207-012830](https://github.com/user-attachments/assets/8af2c989-9697-423d-b7d1-72e3e226cba8)

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.30.4

### Which browsers are you seeing this problem on?

Chrome, Firefox

**Comment by Vanshjain2701** [comment]
Can you please assign this issue to me?

**Comment by Adarsh-Dhar** [comment]
Hey I didn't understand the problem, in the screenshot you provided I can see the grids just fine.

**Comment by Nekzuris** [comment]
> Can you please assign this issue to me?

I can't, only collaborators can.

> Hey I didn't understand the problem, in the screenshot you provided I can see the grids just fine.

The problem is that I’m not supposed to see the grid.

**Comment by tyrasd** [comment]
Some first observations: The transparency of the tiles of the `IGNF_ELEVATION` layer seem to play a role here. Perhaps the tiles slightly overlap at the border.

On the osm.org tile layer, the grid lines only appear to show up during the transition between zoom levels while the new tiles are loading. Is that correct?

**Comment by Nekzuris** [comment]
> Perhaps the tiles slightly overlap at the border.

Yes indeed I found that there is a scale factor of 1.002, making them 256.512px.
It comes from [here](https://github.com/openstreetmap/iD/blob/04fd2ba6f4ca7d0ac7e5f0ca83ac7a1a765e2d20/modules/renderer/tile_layer.js#L21) #3053, I set it to 0 and the border on `IGNF_ELEVATION` is gone, but the seams is back on Chrome, no seams on Firefox.

> On the osm.org tile layer, the grid lines only appear to show up during the transition between zoom levels while the new tiles are loading. Is that correct?

On Firefox yes, on Chrome the grid lines often stay visible after zoom.
